src/statements/CassandraStatements.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/** Create an INSERT prepared statement object.
+ * 
+ * Preparing queries gets the best performance and will ensure that
+ * the JavaScript parameters are correctly mapped to Cassandra types.
+ * 
+ * http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * This will be used for the batch() method, so the object needs to have
+ * two properties: query and params
+ * 
+ * @param {{keyword: string, lang_code: string, ?translations: map<string,string>}}
+ * @returns {{mutation: string, params: Array<string|map<string,string>>}}
+ */
+function prepareInsertTopic(topic) {
```

Comment:
See my [comment](https://github.com/CatalystCode/project-fortis-services/pull/24#discussion_r124914826) on a previous review: I don't think it makes sense to have a utility module for the prepared statements since most of them won't get re-used across resolvers.